components/HomePage/12Contribute/index.jsx

```diff
@@ -21,7 +21,12 @@ const SectionContribute = ({ isNavigationOpen }) => (
       />
 
       <div className="action-btns">
-        <Button hasArrowSuffix type="purple" title="Start Contributing" onClick={() => window.open('https://contribute.autonolas.network')} />
+        <Button
+          hasArrowSuffix
+          type="purple"
+          title="Start Contributing"
+          onClick={() => window.open('https://contribute.autonolas.network')}
```

Comment:
What's the rationale for doing links like this? It feels terrible for accessibility, and I would suspect we are penalised by Google for this.

Reply:
Yes, please create a ticket. We should address our use of <a>, <Link> and Buttons in that. I would make this the next thing we prioritise because this makes it difficult to work with the code currently, and not a great UX. Please create the ticket, and create a plan to address the issue for me to look at before starting work. It would be good if you could do an audit and present an analysis of all the different cases we currently have before proposing a solution.